src/world/area_end/end_00/end_00_3_parade_setup.c

```diff
@@ -1151,7 +1151,11 @@ EvtScript N(EVS_ManageNpcPool) = {
 };
 
 EvtScript N(EVS_ParadePhase_PlayCredits) = {
+#if VERSION_PAL
+    EVT_WAIT(150)
```

Comment:
Maybe this should be 180 * DT. It's been reduced both to accommodate the lower framerate and to make room for more credits.